flow360/plugins/report/uvf_shutter.py

```diff
@@ -572,56 +573,79 @@ class Shutter(Flow360BaseModel):
     access_token: Optional[str] = None
 
     async def _get_3d_images(self, screenshots: dict[str, Tuple]) -> dict[str, list]:
-        @backoff.on_exception(lambda: backoff.constant(3), Flow360WebNotAvailableError, max_time=3600)
-        @http_interceptor
-        async def _get_image_sequence(
-            session: aiohttp.client.ClientSession, url: str, shutter_request: list[dict]
-        ) -> str:
-            log.debug(
-                f"sending request to uvf-shutter: {url=}, {type(shutter_request)=}, {len(json.dumps(shutter_request))=}"
-            )
-            return session.post(url, json=shutter_request)
-
-        async with aiohttp.ClientSession(
-            timeout=aiohttp.ClientTimeout(total=3600),
-            headers={"Authorization": f"Bearer {self.access_token}"},
-        ) as session:
-            tasks = []
-            for _, _, shutter_request in screenshots:
-                tasks.append(
-                    _get_image_sequence(
-                        session=session,
-                        url=urljoin(self.url, "/sequence/run"),
-                        shutter_request=shutter_request,
-                    )
-                )
-                log.debug(
-                    f"request to shutter: {json.dumps(shutter_request)}"
+
+        async def _build_image_from_local(uvf_request: list[dict], output_folder: str) -> list[str]:
```

Comment:
This is pretty big can we break this up.  Also this is async so does that mean we will spin up multiple containers?  Don't we need to set some kind of limit on the number of containers we would spin up?  If that limit already exists I just didn't see it.

Reply:
@MatthewSwords any suggestion for the appropriate lines for each function? 
the function has around 60 lines including all comments. 
I just get the move files to separated one